control-plane/p4RuntimeSymbolTable.cpp

```diff
@@ -254,7 +254,7 @@ void P4::ControlPlaneAPI::P4RuntimeSymbolTable::computeIdsForSymbols(P4RuntimeSy
     for (const auto &mapping : nameToIteratorMap) {
         const cstring name = mapping.first;
         const auto iterator = mapping.second;
-        const uint32_t nameId = jenkinsOneAtATimeHash(name.c_str(), name.size());
+        const uint32_t nameId = Util::Hash{}(name);
```

Comment:
I think it is a good idea to clean this up but a lot of infrastructure relies on the computation of the P4Runtime IDs. The test failures stem from this change because the reference files have changed. I am not sure whether we can update this hash. If we do, we should do it in a separate PR. 

@smolkaj @jonathan-dilorenzo

Reply:
Let me move this to a separate PR